frame/treasury/src/lib.rs

```diff
@@ -111,6 +121,12 @@ type BalanceOf<T> = <<T as Trait>::Currency as Currency<<T as frame_system::Trai
 type PositiveImbalanceOf<T> = <<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::PositiveImbalance;
 type NegativeImbalanceOf<T> = <<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::NegativeImbalance;
 
+/// The treasury's module id, used for deriving its sovereign account ID.
+const MODULE_ID: ModuleId = ModuleId(*b"py/trsry");
+
```

Comment:
Why did you bring this back?

It is now a configuration trait

Reply:
Must be merge error